app/src/main/java/com/example/sharingang/items/Item.kt

```diff
@@ -18,8 +18,7 @@ data class Item(
     /** URLs of the images */
     val images: List<String> = listOf(),
 
-    /** Price in cents */
-    val price: Int = 0,
+    val price: Double = 0.0,
```

Comment:
I think the price being an int and using cents was intentional, I don't know if we should change that or not (some currencies don't have the notion of "cents" or commas used in their money). Maybe check with @betrisey why it was an int before.

Reply:
We discussed it with @betrisey and agreed on the fact that a `double` should be fine. The reason for using an `int` (as cents) was to avoid problems with floating point arithmetic. A price will only be set or read, therefore no floating point arithmetic issues should occur.